readme.txt

```diff
@@ -333,6 +333,7 @@ Please see the changelog for the complete list of changes in this release. Remem
 * Fix - Fixed start and end date limit parsing for events archive in the REST API code [78375]
 * Fix - Fixed Post ID not being sent to the_title filter for Organizers and Venues (props Anna L.) [85206]
 * Fix - Fixed an issue that would sometimes render Event Aggregator options invalid even with a valid license [78469]
+* Fix — Fixed an issue where the mobile.php template file would often fail to include an event's featured image [74291]
```

Comment:
@ggwicz - unicorns are executed every time you use these fancy dashes and break consistency